proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+
+## Proposal
+With repositories under the http://github.com/OpenContainers organization:
+* Rename the OCITools repository as runtime-tools
+* Create a new repository named image-tools and populate this with the files from: https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool
+
+### Project Descriptions
+Below are brief draft project descriptions intended to serve as an initial abstract for each project:
+* Runtime-Tools: Tools for testing of container runtimes implementing the OCI runtime-spec.  
+This includes code that tests a runtime's conformance to the OCI Container Runtime spec.
+* Image-Tools: Tools for testing of container images implementing the OCI image-spec.  
+This includes code that validates a file's conformance to the OCI Container Image Format spec.  
+
+### Initial Maintainers
+Initial maintainers of the runtime-tools project:
+* Maintainers of the runtime-spec project
+* Three additional most active contributors (by commit count as of Wed, 8/24/2016) to the repo that will become runtime-tools:
+  - Ma Shimiao (https://github.com/Mashimiao)
+  - W. Trevor King (https://github.com/wking)
+  - Liang Chenye (https://github.com/liangchenye)
+
+(Murnal Patel is one of the most active contributors and he will be included with the first bullet point.)
+
+Initial maintainers of the image-tools project shall be:
+* Maintainers of the image-spec project
+* Three additional most active contributors (by commit count as of Wed, 8/24/2016) to the files that will become image-tools:
+  - Sergiusz Urbaniak (https://github.com/s-urbaniak)
+  - Antonio Murdaca (https://github.com/runcom)
+  - xiekeyang (https://github.com/xiekeyang)
+
+(Vincent Batts and Jonathan Boulle are two of the most active contributors and they will be included with the first bullet point.)
+
+### Code of Conduct
+Both of the proposed projects would incorporate (by reference) the OCI Code of Conduct
+
+### Governance and Releases
+Both of the proposed projects would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+Released version numbers of the runtime-tools and image-tools projects should roughly align with released verions of the associated specs projects.
+
+It is expected that releases of the tools projects will follow releases of the specs projects by anywhere from a few days to a few months.
+
+## Frequenty Asked Questions (FAQ)
+Q: What about the runc project?
```

Comment:
I personally prefer PascalCase or lowerCamelCase for readability, but the [runc project readme ](https://github.com/opencontainers/runc/blob/master/README.md) uses alllowercase.